### ISSUE TITLE
Option to disable your mod Player render and animations when using Epic Fight

### ISSUE THREAD

**Issue by maximogta** [Skullman-G/MC-Dark-Souls#31]
Option to disable your mod Player render and animations when using Epic Fight, 

I really like the mod, but just want to use the Epic fight mod player animations and disable yours, would be a nice option for mod pack makers too.

Or make the necessary animations like miracle summon a addon for epic fight.
Thank you.

**Comment by Skullman-G** [comment]
Okay. So I will definitely add an option to disable the player combat in the ingame config, but I don't think I will make it as an addon for epic fight.

**Comment by maximogta** [comment]
Thank you very much

**Comment by Skullman-G** [comment]
Oh no. That also means that I have to remove the Falconer armor because the epic fight mod doesn't have the model for it.

**Comment by Skullman-G** [comment]
And because there are many more things that would need to be removed or extremely altered I decided to don't do it. I hope you can understand my choice. There has to be some other way to make this mod more configurable besides full on disabling the core mechanics. I could make an option to add armor sets. And there will certainly be new weapon types. Also I am working on making the combat animations more fluid and faster.

**Comment by Skullman-G** [comment]
Wait. You said you just wanted the animations of epic fight mod, right? So how about I add configurable weapon types? That way everyone can add animations as .dae files and register them in the server config file.

**Comment by maximogta** [comment]
For me would be better if I could use epic fight mod because it has datapack support, so mobs from other mods can use epic fight animations.
Maeby your aproach would be even better, giving the possibility for players to create custom animations for other mobs. 
Could end up being a really cool idea. 
Is your mod so you decide.

**Comment by XiFengXF** [comment]
Has this function been realized now? 
I like the attributes of the dark soul, such as physical strength, weight, block attribute, UI, etc. However, the animation of epic fight in biological movements is smoother and more in line with public preferences.
If the two can complement each other, it is indeed a very good decision!

**Comment by Skullman-G** [comment]
No. We will improve on our own combat system instead of making it compatible to others.
